guides/v2.3/graphql/develop/resolvers.md

```diff
@@ -194,7 +194,7 @@ mutation {
 
 A sample response:
 
-```text
+```json
```

Comment:
I think should be as text. @keharper right?

Reply:
GraphQL requests should be `text`. The responses should be `json`.  Originally, we had both as `json`, then overcorrected by setting both to `text`. This is the correct way for now.